.github/workflows/zowe-cli-plugin.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```

Comment:
now that npm8 is bundled with the remaining version of node, I believe we can remove the "update npm" step